src/styles/components/_detailsBanner.scss

```diff
@@ -1,12 +1,16 @@
 .details-banner {
-  background-position: center;
-  background-repeat: no-repeat;
-  background-size: cover;
   line-height: 2.125rem; // 28px
   letter-spacing: 0.02em; // 2% in Figma
+  max-width: 82.875rem; // compensates for 8.5625rem padding on left/right
 
-  @media (min-width: $breakpoint-large-screen-860) {
-    margin: -8.5625rem;
+  @media only screen and (min-width: 100rem) {
+    margin: auto;
+  }
+
+  @media (min-width: $breakpoint-scale-up) {
+    // margin: -8.5625rem;
+    font-size: map.get($font-sizes, "xl");
+    line-height: 4.1875rem; // 67px/16
```

Comment:
Line-height is a *unitless* property. You should not specify rems, ems, pixels, etc. You want line-height to be relative to the font size of the element's font size. 

So if your xl font is 3.125rem (50px - also why are we doing base 10 font sizes and not base 4? Makes math weird).
And you want your line height to be 4.1875 rem (67px).

You want to use `line-height: 1.34`. The 1.34 comes from 67/50 or 3.125/4.1875. 

Though I'd round it to 1.4. The most common values for line-height are 1, 1.1, and 1.4. 

I've probably sent you this before, [but I have a video talking about some of these things with Z](https://www.youtube.com/watch?v=XoB1fSRCBng) on our Youtube account

Reply:
-Now that you say that, it does sound familiar that line-height is unitless. Someone probably mentioned it in a different PR. Thanks for reminding me!

-Changed to 1.4 for now only in this file. I think refactoring all of our line-heights properly will need to be a new card again. https://github.com/sparkbox/accessible-components/pull/89/commits/5caa631fbe674d5c7322e967e502754ce8687642

-Thanks for that video! I have not seen this yet :)